src/test/java/K_pages/HomePage.java

```diff
@@ -0,0 +1,24 @@
+package K_pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage extends BasePage{
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )